src/components/toolbar/TooltipTemplate.tsx

```diff
@@ -0,0 +1,184 @@
+/** @jsxImportSource @emotion/react */
+import { SerializedStyles, css } from '@emotion/react';
+import { CSSProperties } from 'react';
+
+const shortcutBoxSize = 1.5;
+const color: CSSProperties['color'] = 'white';
+
+const styles: Record<
+  | 'titleContainer'
+  | 'title'
+  | 'description'
+  | 'shortcutContainer'
+  | 'shortcutItem',
+  CSSProperties
+> &
+  Record<'subTitleListItem' | 'subTitleList', SerializedStyles> = {
```

Comment:
I wrote some guidelines on how to style things in react-science. Please have a look at https://github.com/zakodium-oss/react-science/wiki/Contributor-guidelines#styling-components

Good example in react-science: https://github.com/zakodium-oss/react-science/blob/main/src/components/drop-zone/DropZone.tsx

The guidelines are and we can update them if more usage patterns are needed.